src/components/chart/ha-chart-base.ts

```diff
@@ -288,6 +329,25 @@ export class HaChartBase extends LitElement {
           display: false,
         },
       },
+      {
+        id: "labelClickHandler",
```

Comment:
This should not be in `ha-chart-base`, use the `plugin` property to add a plugin to the chart from just the place you need it.

See for example:

https://github.com/home-assistant/frontend/blob/dev/src/panels/lovelace/cards/energy/hui-energy-devices-graph-card.ts#L156-L166